projects/addon-charts/components/bar-chart/bar-chart.template.html

```diff
@@ -24,6 +25,7 @@
             [size]="size"
             [tabIndex]="hintContent ? 0 : -1"
             [value]="set"
+            (keydown.enter)="tapColumn.emit(index)"
```

Comment:
nit: I think it bubbles, so it can be moved to the same place as click

Reply:
fixed